stacks.yml

```diff
@@ -543,6 +543,17 @@ jenkins_ssh_access:
     ToPort: 22
     SecurityGroup: "{{ stacks.jenkins.outputs.InstanceSecurityGroup }}"
 
+jenkins_letsencrypt_access:
```

Comment:
I'm not sure I follow what this is for - is this to get another new cert?  And then we kill it once they have verified the server and issued the cert to us?  Would we then need to turn it back on for renewal in a couple of years time?   Or have I misunderstood?